splink/linker.py

```diff
@@ -848,6 +848,47 @@ def load_settings(self, settings_dict: dict | str):
         self._validate_input_dfs()
         self._validate_dialect()
 
+    def initialise_settings(self, settings_dict: dict):
+        """*This method is now deprecated. Please use `load_settings`
+        when loading existing settings or a pre-trained model.*
+
+        Initialise settings for the linker.  To be used if settings were
+        not passed to the linker on creation.
+        Examples:
+            >>> linker = DuckDBLinker(df, connection=":memory:")
+            >>> linker.profile_columns(["first_name", "surname"])
+            >>> linker.initialise_settings(settings_dict)
+        Args:
+            settings_dict (dict): A Splink settings dictionary
+        """
+        # If a uid already exists in your settings object, prioritise this
+        settings_dict["linker_uid"] = settings_dict.get("linker_uid", self._cache_uid)
+        settings_dict["sql_dialect"] = settings_dict.get(
+            "sql_dialect", self._sql_dialect
+        )
+        self._settings_dict = settings_dict
+        self._settings_obj_ = Settings(settings_dict)
+        self._validate_input_dfs()
+        self._validate_dialect()
+        logger.log(
```

Comment:
what about `warnings.warn()` instead, it might be more visible?

Or could bring in in @deprecated decorator from https://pypi.org/project/Deprecated/